src/spacel/provision/orbit/gdh.py

```diff
@@ -94,6 +95,7 @@ def _orbit_from_child(orbit, region, name, cf_parameters, cf_outputs):
         orbit._public_elb_subnets[region] = public_subnets
         orbit._private_instance_subnets[region] = private_subnets
         orbit._private_elb_subnets[region] = private_subnets
+        # FIXME: DbSubnetGroups: private_cache, private_rds, public_rds
```

Comment:
Nope; FIXME is blocked by upstream.

I expect the `OrbitFactory` to return cache/public-rds/private-rds subnet groups, but GitDeploy doesn't build those (so `GitDeployOrbitFactory` can't find it in outputs).

I _could_ shim in a template to just turn the 6x subnets into 3x subnet groups, but the better(?) plan was to just have GDH build the subnet groups in it's own `vpc.template`: either in new subnets, or re-using existing subnets as I've been doing in all the manual setups.

For now you can only use caches (and later: RDS) on "pure" spacel orbits: planning for the backport to GDH with breadcrumbs like this, but not doing that part for free :P